Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Author Cameron-git
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive 
+
+WORKDIR /app
+
+RUN apt-get update
+RUN apt-get upgrade -y
+
+RUN apt-get install -y git python3-pip
+RUN apt-get install -y gcc-10 g++-10
+
+# Pytorch
+RUN pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu116
+
+# CUDA 11.6
+RUN apt-get install -y wget zlib1g
+RUN apt-key del 7fa2af80
+RUN wget https://developer.download.nvidia.com/compute/cuda/repos/wsl-ubuntu/x86_64/cuda-keyring_1.0-1_all.deb
+RUN dpkg -i cuda-keyring_1.0-1_all.deb
+RUN apt-get update
+RUN apt-get install -y cuda-toolkit-12-0
+ENV PATH="/usr/local/cuda/bin:$PATH"
+
+# CUDNN 8.7
```

Comment:
Dont need cudnn anymore

Reply:
True, I'll remove that section now, but I would suggest we fix the container in a separate PR: https://github.com/norse/aestream/issues/53